pkgs/development/python-modules/gpgme/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  automake,
+  buildPythonPackage,
+  gpgme,
+  lib,
+  setuptools,
+  swig,
+}:
+
+buildPythonPackage rec {
+  pname = "gpgme";
+  inherit (gpgme) version src patches;
+  pyproject = true;
+
+  postPatch = ''
+    substituteInPlace lang/python/setup.py.in \
+      --replace "gpgme_h = '''" "gpgme_h = '${lib.getDev gpgme}/include/gpgme.h'"
+  '';
+
+  configureFlags = gpgme.configureFlags ++ [
+    "--enable-languages=python"
+  ];
+
+  postConfigure = "
+    cd lang/python
+  ";
+
+  preBuild = ''
+    make copystamp
+  '';
+
+  build-system = [ setuptools ];
+
+  nativeBuildInputs = [
+    swig
+  ];
+
+  buildInputs = [
+    gpgme
+  ];
+
+  pythonImportsCheck = [ "gpg" ];
+
+  meta = {
+    description = "Python bindings to the GPGME API of the GnuPG cryptography library";
+    homepage = "https://dev.gnupg.org/source/gpgme/browse/master/lang/python/";
+    inherit (gpgme.meta) changelog license maintainers;
```

Comment:
We should merge all of meta to not forget anything, right?

Reply:
I thought that creates problems with `meta.position` but it must be some other problem.

Reply:
@eclairevoyant Can you explain why I get
```
nix-repl> python3.pkgs.gpgme.meta.position
"nixpkgs/pkgs/development/python-modules/gpgme/default.nix:45"
```
here despite using `meta = gpgme.meta // ...`? C.f. https://github.com/NixOS/nixpkgs/pull/324127#discussion_r1712991684

Reply:
It's because, not despite.

Reply:
But removing `gpgme.meta //` yields the same `meta.position`.